app/src/main/java/com/mapbox/maps/testapp/examples/terrain/SantaCatalinaActivity.kt

```diff
@@ -61,7 +61,7 @@ class SantaCatalinaActivity : AppCompatActivity() {
       style(styleUri = Style.SATELLITE_STREETS) {
         +rasterDemSource(SOURCE) {
           url(TERRAIN_URL_TILE_RESOURCE)
-          tileSize(512)
+          tileSize(514)
```

Comment:
Just curious, why change to 514?

Reply:
More context at https://github.com/mapbox/mapbox-maps-ios/issues/470, I will also add a in-line comment here.

Reply:
```suggestion
          // 514 specifies padded DEM tile and provides better performance than 512 tiles.
          tileSize(514)
```